packaging/debian-sid/changelog

```diff
@@ -1,3 +1,11 @@
+snapd (2.59.4-1) unstable; urgency=medium
+
+  * New upstream release, LP: #2009946
+    - Retry when looking label on non-UEFI systems
```

Comment:
I think this should mention https://bugs.launchpad.net/snapd/+bug/2018977

Reply:
I updated this to include the LP bug now